microsoft_outlook_single_tenant/tests/test_fetchmail_outlook.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo.tests.common import SavepointCase
+
+
+class TestMicrosiftOutlook(SavepointCase):
+    def test_single_tenant_app_endpoint(self):
+        settings = self.env["res.config.settings"].create({})
+        settings.microsoft_outlook_client_identifier = "test_client_id"
+        settings.microsoft_outlook_tenant_id = "test_tenant_id"
+        settings.microsoft_outlook_client_secret = "test_secret"
+        settings.execute()
+
+        mail_server = self.env["fetchmail.server"].create(
+            {
+                "name": "Test server",
+                "use_microsoft_outlook_service": True,
+                "user": "test@example.com",
+                "password": "",
+                "server_type": "imap",
+                "is_ssl": True,
+            }
+        )
+
+        single_tenant_endpoint = (
+            "https://login.microsoftonline.com/test_tenant_id/oauth2/v2.0/authorize"
```

Comment:
```suggestion
            "https://login.microsoftonline.com/test_tenant_id/oauth2/v2.0/"
```

As the `authorize` is added by computation during the core process, you should not add it.